azure_arc_servers_jumpstart/docs/arc_inventory_tagging.md

```diff
@@ -0,0 +1,119 @@
+* [GCP Ubuntu VM](../docs/gcp_terraform_ubuntu.md)
+* [AWS Ubuntu VM](../docs/aws_terraform_ubntu.md)
+
+# Prerequisites
+
+* Clone this repo
+
+* [Install or update Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest). **Azure CLI should be running version 2.7** or later. Use ```az --version``` to check your current installed version.
+
+* Resources projected into Azure via Azure Arc from AWS and/or GCP.
+
+# Verify that your Arc connected servers are ready for tagging
+
+We will be using Resource Graph Explorer during this exercise to query and view resources in Azure. 
+
+* Enter "Resource Graph Explorer" in the top search bar in the Azure portal and select it.
+
+    ![](../img/inventory/01.png)
+
+* In the query window, enter the following query and then click "Run Query":
+
+        Resources
+        | where type =~ 'Microsoft.HybridCompute/machines'
+
+* If you have correctly connected servers via Arc, they should be listed in the Results pane of Resource Graph Explorer.
+
+    ![](../img/inventory/02.png)
+
+# Create a basic Azure tag taxonomy
+
+* Open AZ CLI and run the following commands to create a basic taxonomy structure.
+
+    ```bash
+    az tag create --name "Hosting Platform"
+    az tag add-value --name "Hosting Platform" --value "Azure"
+    az tag add-value --name "Hosting Platform" --value "AWS"
+    az tag add-value --name "Hosting Platform" --value "GCP"
+    az tag add-value --name "Hosting Platform" --value "On-premises"
+    ```
+
+    ![](../img/inventory/05.png)
+
+# Tag Arc resources
+
+Now that we have created a basic taxonomy structure, we can apply tags to our Arc server resources. In this guide, we will demonstrate tagging resources in both AWS and GCP. If you only have resources in one of these providers, you can skip to the appropriate section for AWS or GCP.
+
+## Tag Arc-connected AWS Ubuntu server
+
+* In AZ CLI, run the following commands to apply the "Hosting Platform : AWS" tag to your Arc AWS servers. 
+
+    *Note: If you connected your AWS servers using a method other than the one described in [this tutorial](../docs/aws_terraform_ubuntu.md), then you will need to adjust the values for `awsResourceGroup` and `awsMachineName` to match values specific to your environment.*
+
+    ```bash
+    export awsResourceGroup="arc-aws-demo"
+    export awsMachineName="arc-aws-demo"
+    export awsMachineResourceId="$(az resource show --resource-group $awsResourceGroup --name $awsMachineName --resource-type "Microsoft.HybridCompute/machines" --query id)"
+    export awsMachineResourceId="$(echo $awsMachineResourceId | tr -d "\"" | tr -d '\r')"
+    az resource tag --ids $awsMachineResourceId --tags "Hosting Platform"="AWS"
+    ```
+
+    ![](../img/inventory/07.png)
+
+## Tag Arc-connected GCP server
```

Comment:
I would add a screenshot showing the tag on the resource in the portal. It can make people relate a visual manner.